lisa/src/test/java/it/unive/lisa/test/imp/tutorial/Sign.java

```diff
@@ -0,0 +1,296 @@
+package it.unive.lisa.test.imp.tutorial;
+
+import it.unive.lisa.analysis.SemanticDomain.Satisfiability;
+import it.unive.lisa.analysis.SemanticException;
+import it.unive.lisa.analysis.nonrelational.BaseNonRelationalValueDomain;
+import it.unive.lisa.cfg.type.Type;
+import it.unive.lisa.symbolic.value.BinaryOperator;
+import it.unive.lisa.symbolic.value.Constant;
+import it.unive.lisa.symbolic.value.Identifier;
+import it.unive.lisa.symbolic.value.TernaryOperator;
+import it.unive.lisa.symbolic.value.UnaryOperator;
+
+public class Sign extends BaseNonRelationalValueDomain<Sign> {
```

Comment:
Same considerations as for the `Parity` class: singletons for top, bottom, zero, neg and pos, no need to test for equality in `lubAux` and `lessOrEquals`, consider also `long`, `float` and `double`.